Fun Commands/Mini-Games/Dyno-RPG/blight.txt

```diff
@@ -0,0 +1,15 @@
+{delete}
+{silent}
+{require:{&Adventurer}}
+{!role $1 +Blighted}
+**{user.username}** has cast Blight on **$1**!
+
+The sound of buzzing can be heard in the distance. **$1** look up to see what it is. 
```

Comment:
Little spelling thing, might want it to be "looks" up

Reply:
fixed